### PR TITLE
#158 Allowlist of Ruuter services exposed as public REST endpoints

### PR DESCRIPTION
**Pull request by RayDNoper**
Added functionality to limit access to specific internal DSL's to specific IPs and referrers.

### CODE REVIEW

File: samples/CONFIGURATION.md
Comment (turnerrainer) [inline samples/CONFIGURATION.md:162]:
```suggestion
Some DSL services can be exposed to internal requests as REST endpoints.
```

File: samples/CONFIGURATION.md
Comment (turnerrainer) [inline samples/CONFIGURATION.md:170]: Do these work when using ports as well? For example, `127.0.0.1:8081` and `http://localhost:8082/internalTest`.

File: samples/CONFIGURATION.md
Comment (turnerrainer) [inline samples/CONFIGURATION.md:163]: Specified in `allowedFiletypes` or `internalRequests`?

**Review by turnerrainer — CHANGES_REQUESTED** [review]
Some comments need addressing before merging.

File: samples/CONFIGURATION.md
Comment (RayDNoper) [reply samples/CONFIGURATION.md:170]: For IPs, the port can not be specified (as outgoing request port can vary). For referrer URL, the port can be specified.

File: samples/CONFIGURATION.md
Comment (RayDNoper) [reply samples/CONFIGURATION.md]: Fixed typo.

**Comment by RayDNoper** [comment]
As I cannot review my own PR, merged without formal approval after addressing the  questions.

**Pull request merged**
